packages/amplify-category-api/resources/awscloudformation/overrides-resource/AppSync/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "overrides",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w"
+  },
+  "dependencies": {
+    "@aws-amplify/cli-extensibility-helper": "2.3.0-apiext3.0"
```

Comment:
same here

Reply:
Yes, should be "^2.0.0" like other categories (auth and storage).

Reply:
We haven't published a version with API types included. A tagged version is necessary on the first release.

Reply:
But, when we publish to prod/release ,"@aws-amplify/cli-extensibility-helper" will be published will API types.

Reply:
That's a good point, then it should be ^2.3.0, correct?

Reply:
it should be whatever is in the package.json of cli-extensibility-helper. learna will do the rest

Reply:
@edwardfoyle Will lerna make changes if these pacakge.json's are in the `...category-api/resources/` directory?